packages/cli/src/lib/machine/drivers/azure/types.ts

```diff
@@ -0,0 +1,15 @@
+type RootObjectDetailsError = {
```

Comment:
a file named `types` is an anti-pattern - just like you don't have files named `objects` or `classes`. put this code together with non-types where it belongs feature-wise.